accounts/models.py

```diff
@@ -118,6 +116,65 @@ def locations(self):
             )
         )
 
+    def email_not_disabled(self, email_type_name):
+        # Raise exception if the email_type doesn't exists
+        email_type = EmailPreferenceType.objects.get(name=email_type_name)
+
+        # when send_by_default == invert_default means email is disabled
+        invert_default = UserEmailSetting.objects.filter(user=self.user,
+                email_type=email_type).count()
```

Comment:
Would be nicer to use exists() rather than count: https://docs.djangoproject.com/en/1.9/ref/models/querysets/#exists